builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,22 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+
+	local count = 1
+	for i = 9, 1, -1 do
+		local check_path = path .. "." .. i
+		local file = io.open(check_path,"rb")
+		if file ~= nil then
+			count = i
+			io.close(file)
+			break
+		end
+	end
+	local new_path = path .. "." .. math.random(1, count)
+	if math.random(1, (count+1)) ~= 1 then
```

Comment:
Because without that, either background.png or background.png.1 will not be shown (depending on other changes made), which, imho, is not ideal behaviour.

The first one decides the image, with the among being the number of background.png.x found, while the second one decides whether or not to display background.png, giving it the same chance as the other images of being displayed.